go.mod

```diff
@@ -1,10 +1,30 @@
 module github.com/reddit/edgecontext
 
-go 1.16
+go 1.17
 
 require (
-	github.com/apache/thrift v0.14.1
+	github.com/apache/thrift v0.15.0
```

Comment:
ideally we want to upgrade thrift compiler to 0.15.0 as well and update the thrift compiled go and py files. but I _think_ currently baseplate.py is still using 0.14.x library (not sure which point version).

as far as I know there's no breaking changes in thrift compiler, for both py and go, between 0.14.0 and 0.15.0, so it's ok to use go 0.15.0 library with 0.14.x compiler, and it's also ok to use py 0.14.x library with 0.15.0 compiler, so I don't feel strongly either way, but I do think baseplate.py (and edgecontext py library) should upgrade thrift library dependency to 0.15.0 sooner rather than later. cc @spladug